app/data_models/progress_store.py

```diff
@@ -77,12 +77,11 @@ def is_routing_backwards(self) -> bool:
     def is_section_complete(self, section_key: SectionKey) -> bool:
         """
         Return True if the CompletionStatus of the Section or List Item specified by the given section_id and
-         list_item_id is COMPLETED or INDIVIDUAL_RESPONSE_REQUESTED, else False.
+         list_item_id is COMPLETED, else False.
         """
         return section_key in self.section_keys(
```

Comment:
Since there are no multiple statuses now, the whole logic can be simplified here (so you don't look for an item(status) in a sequence of statuses, instead check if status is something, in this case, COMPLETED).